### PR TITLE
BUG/MINOR: Correctly hanlde IPv6 addresses when parsing <ip>:<port> pair

### PR DESCRIPTION
**Pull request by daa**
Currently parser split <ip>:<port> pair by the first colon which leads to incorrect result in case of IPv6, this pull request interprets the part after the last colon as a port and everything before as an address.

Closes: #50

### CODE REVIEW

**Comment by oktalz** [comment]
hi @daa 
PR looks good
it seems you commit subject message is a bit to long for our standards

subject word count out of bounds [words 3 < 16 < 15] 
`Split <ip>:<port> pair by the last colon instead of the first to work correctly with IPv6`

could you rephrase it ?

btw, this is only a limit for commit message title, body of a commit message can be any size

also, if you want, I can change it to comply with the CI.

**Comment by oktalz** [comment]
@daa we have a policy to have a clean git tree, 

basically without [Merge branch 'haproxytech:master' into correct-ip-port-split](https://github.com/haproxytech/config-parser/pull/51/commits/091c5c551bc342506315045eeb18728444f8a60e)

so if you can resolve that in PR to only have commits that points to HEAD,
or if it seems like a lot of work I can cherry pick them and merge them without PR

**Comment by daa** [comment]
@oktalz Thank you for comment, rebased branch.
